src/ui.form-select/USelect.vue

```diff
@@ -585,12 +87,17 @@ const isOpen = ref(false);
 const preferredOpenDirection = ref(DIRECTION.bottom);
 const search = ref("");
 
-const dropdownListRef = ref(null);
-const wrapperRef = ref(null);
-const searchInputRef = ref(null);
-const labelComponentRef = ref(null);
-const leftSlotWrapperRef = ref(null);
-const innerWrapperRef = ref(null);
+const dropdownListRef = ref<DropdownListRef | null>(null);
```

Comment:
You don't need custom DropdownListRef type, detete it and use [ComponentExposed](https://github.com/vuelessjs/vueless/blob/main/src/types.ts#L477) utility type. Look for examples in other components.